### PR TITLE
Improve boot.js pready for some optimising

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

- makes the syntax changes of #19007 without adding the commons stuff
- kicks off loading enhanced sooner

## What is the value of this and can you measure success?

preperation for some further improvments, clarity to enable it, but keeping PRs small

### CODE REVIEW

File: static/src/javascripts/boot.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const go = () => {
         bootStandard();
 
         // 2. once standard is done, next is commercial
-        if (config.page.isDev) {
+        if (process.env.NODE_ENV !== 'production') {
```
Comment (sndrs) [inline static/src/javascripts/boot.js]: let's just not serve this code in PROD

**Review by SiAdcock — APPROVED** [review]
Good yak shaving 🍧

**Review by rich-nguyen — APPROVED** [review]
super

**Comment by regiskuckaertz** [comment]
Oooh 😍 can this be applied in the atoms bootstrap module too? (I can do it later, just asking)

**Comment by sndrs** [comment]
@regiskuckaertz to refactor using import? yeah should be – i'll merge this today then go for it

**Pull request merged**